test/Microsoft.ML.Tests/Scenarios/Api/TestApi.cs

```diff
@@ -289,5 +289,52 @@ private List<BreastCancerExample> ReadBreastCancerExamples()
                 .ToList();
             return data;
         }
+
+        [Fact]
+        public void TestTrainTestSplit()
+        {
+            var mlContext = new MLContext(0);
+            var dataPath = GetDataPath("adult.tiny.with-schema.txt");
+            // Create the reader: define the data columns and where to find them in the text file.
+            var input = mlContext.Data.ReadFromTextFile(dataPath, new[] {
+                            new TextLoader.Column("Label", DataKind.BL, 0),
+                            new TextLoader.Column("Workclass", DataKind.TX, 1),
+                            new TextLoader.Column("Education", DataKind.TX,2),
+                            new TextLoader.Column("Age", DataKind.R4,9)
+            }, hasHeader: true);
+
+            var (simpleTrain, simpleTest) = mlContext.BinaryClassification.TrainTestSplit(input);
+            Func<IDataView, List<string>> getWorkclass = (IDataView view) =>
+            {
+                return view.GetColumn<ReadOnlyMemory<char>>(mlContext, "Workclass").Select(x => x.ToString()).ToList();
+            };
+            var simpleTrainWorkclass = getWorkclass(simpleTrain);
+            var simpleTestWorkClass = getWorkclass(simpleTest);
+            var (simpleTrainWithSeed, simpleTestWithSeed) = mlContext.BinaryClassification.TrainTestSplit(input, seed: 10);
+            var simpleWithSeedTrainWorkClass = getWorkclass(simpleTrainWithSeed);
+            var simpleWithSeedTestWorkClass = getWorkclass(simpleTestWithSeed);
+            // Validate we get differnt test sets (they are smaller than TrainSet)
+            Assert.NotEqual(simpleTestWorkClass, simpleWithSeedTestWorkClass);
+
+            var (stratTrain, stratTest) = mlContext.BinaryClassification.TrainTestSplit(input, stratificationColumn: "Workclass");
```

Comment:
Maybe mark these blocks somehow to show that they are testing different parameter combinations.